preupg/xmlgen/oscap_group_xml.py

```diff
@@ -55,33 +55,28 @@ def find_all_ini(self):
         for file_name in self.lists:
             if FileHelper.check_file(file_name, "r") is False:
                 continue
-            try:
-                config = configparser.ConfigParser()
-                filehander = codecs.open(file_name, 'r', encoding=settings.defenc)
-                config.readfp(filehander)
-                fields = {}
-                if config.has_section('premigrate'):
-                    section = 'premigrate'
-                else:
-                    section = 'preupgrade'
-                for option in config.options(section):
-                    fields[option] = config.get(section, option)
-                self.loaded[file_name] = [fields]
-            except configparser.MissingSectionHeaderError:
-                MessageHelper.print_error_msg(title="Missing section header")
-            except configparser.NoSectionError:
-                MessageHelper.print_error_msg(title="Missing section header")
-            except configparser.ParsingError:
-                MessageHelper.print_error_msg(title="Incorrect INI file\n", msg=file_name)
-                os.sys.exit(1)
+            config = configparser.ConfigParser()
+            filehander = codecs.open(file_name, 'r', encoding=settings.defenc)
+            config.readfp(filehander)
+            fields = {}
+            if config.has_section('premigrate'):
+                section = 'premigrate'
+            else:
+                section = 'preupgrade'
+            for option in config.options(section):
+                fields[option] = config.get(section, option)
+            self.loaded[file_name] = [fields]
```

Comment:
I've tried to replicate your ouput and I'm not successful:
```
$ tools/preupg-xccdf-compose /<path>/RHEL6_7
Error: Tag(s) content_title missing in the /<path>/RHEL6_7-results/backup/bacula/module.ini file
```
Please make sure you are running the correct code from this branch.